### DISCUSSION TITLE
Delete original image after upload?

### DISCUSSION THREAD

**Discussion by towbeeWee** [pocketbase/pocketbase#4630] [Q&A] [answered]
Hi, the ability to create multiple thumbnails after uploading is awesome. Is it also possible to delete the original image after the upload and only keep the cropped images? This would save some storage space, as users often upload very large image files which are rarely used in frontend. Thanks in advance

**Comment by ganigeorgiev** [comment, marked as answer]
No, it is not possible because the thumbs are generated lazily on first access from the original image, aka. you need the original image to exist in order to have a PocketBase generated thumb.

Depending your use case an alternative could be to generate the thumb on the client-side and upload the generated thumb file to the PocketBase server.

**Comment by towbeeWee** [comment]
Thanks for the explanation and answer. The possibility of creating thumbnails on the client side is a possibility that I have already partially implemented. If the backend had already implemented the function, I wouldn't need the duplicate code :). Thanks again.
